### ISSUE TITLE
fos_rest.exception.codes configuration bug?

### ISSUE THREAD

**Issue by tael** [FriendsOfSymfony/FOSRestBundle#1571]
Hi, I'm testing `Exception Handling` and stopped by a little problem belows.

I'm not sure this is a bug or NOT. But if not, I wish someone helps me! :D

Symfony\Component\HttpFoundation\Response::HTTP_* constant does not works.

from the manual ( http://symfony.com/doc/master/bundles/FOSRestBundle/4-exception-controller-support.html )
> ...
> To map Exception classes to HTTP response status codes an exception map may be configured, where the keys match a fully qualified class name and the values are either an integer HTTP response status code or a string matching a class constant of the Symfony\Component\HttpFoundation\Response class:
> ...

This Configuration does works, well
```
fos_rest.exception.codes
            'Symfony\Component\Security\Core\Exception\AccessDeniedException': 403
```
But, This configuration does not works

```
fos_rest.exception.codes
            'Symfony\Component\Security\Core\Exception\AccessDeniedException': HTTP_FORBIDDEN
```

**Comment by tael** [comment]
I reviewed source code,
1.5 looks work maybe,
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/1.5/Util/Codes.php
but 2.0 ? 404 not found this class
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/2.0/Util/Codes.php
is it deprecated?

**Comment by mvrhov** [comment]
The symfony does contain all the codes necessary and thus that class is gone in 2.0

**Comment by tael** [comment]
yes I agree.  but HTTP_FORBIDDEN is not works. 
this feature is gone also?

**Comment by edefimov** [comment]
@tael No, it's not gone, it looks like a bug in the bundle. [This PR](https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1585) fixes this issue

**Comment by xabbuh** [comment]
This was fixed by #1585.
